ftw/theming/resources/scss/elements/button.scss

```diff
@@ -0,0 +1,77 @@
+$color-button-default: $color-default !default;
+$color-button-success: $color-success !default;
+$color-button-warning : $color-warning !default;
+$color-button-danger: $color-danger !default;
+
+$font-weight-button: normal !default;
+$line-height-button: $line-height-base !default;
+$font-style-button: normal !default;
+$font-size-button: 1em !default;
+
+@mixin button($color-button: $color-secondary, $disabled: false) {
+  display: inline-block;
+
+  min-height: 1em;
+
+  outline: none;
```

Comment:
This disables the browser's outline, which is needed for accessibility reasons.
It should be kept unless there is a visually good indicator that the button is focused (by tabbing, for example).
I'd prefer keeping the browser default focus because it is consistent with links and looks good (at least in chrome).